setup.py

```diff
@@ -17,6 +17,7 @@
         'click>=6',
         'first',
         'six',
+        'setuptools>=20.2.2',
     ],
```

Comment:
it's bad to depend on setuptools: https://github.com/pypa/setuptools/blob/master/CHANGES.rst#v3400